### PR TITLE
[INTERNAL] Builder: Document minify task and input source maps

### PR DESCRIPTION
**Pull request by RandomByte**
JIRA: CPOUI5FOUNDATION-467

### CODE REVIEW

File: docs/pages/Builder.md [hunk 1]
```diff
@@ -83,6 +83,30 @@ All available standard tasks are documented [in the API reference](https://sap.g
...
+Related to this, the bundling tasks will also incorporate the generated source maps to map the content of the bundles to the individual debug variants of the bundled modules.
+
+#### Input Source Maps
+
+!!! info
+	Support for input source maps has been added in UI5 CLI [`v3.7.0`](https://github.com/SAP/ui5-cli/releases/tag/v3.7.0).
+
+For projects facilitating transpilation (such as TypeScript based projects), it is commonly desired to debug in the browser using the original sources, e.g. TypeScript files. To make this work, first the the transpilation process needs to create source maps and reference them in the generated JavaScript.
+
+The UI5 Tooling `minify` task will then find this reference and incorporate the source map into the minification process. In the end, the minified JavaScript resources will reference an updated source map, which reflects the transpilation as well as the minification. The browser can use this to map every statement back to the original TypeScript file, making debugging a breeze.
```
Comment (KlattG) [inline docs/pages/Builder.md:103]:
```suggestion
UI5 Tooling's `minify` task will then find this reference and incorporate the source map into the minification process. In the end, the minified JavaScript resources will reference an updated source map, which reflects the transpilation as well as the minification. The browser can use this to map every statement back to the original TypeScript file, making debugging a breeze.
```

File: docs/pages/Builder.md [hunk 2]
```diff
@@ -83,6 +83,30 @@ All available standard tasks are documented [in the API reference](https://sap.g
...
+
+Expand the below block to view a diagram illustrating the minification process and source map handling.
```
Comment (KlattG) [inline docs/pages/Builder.md:105]:
```suggestion
Expand the block below to view a diagram illustrating the minification process and source map handling.
```

**Review by KlattG — CHANGES_REQUESTED** [review]

File: docs/images/UI5_Tooling/Task_Minify.svg
Comment (KlattG) [inline docs/images/UI5_Tooling/Task_Minify.svg:1]: I think there's no need to use title case in this graphic.

**Comment by d3xter666** [comment]
LGTM

**Review by matz3 — APPROVED** [review]
LGTM. I didn't check the rendered page though

File: docs/images/UI5_Tooling/Task_Minify.svg
Comment (RandomByte) [reply docs/images/UI5_Tooling/Task_Minify.svg:1]: I would keep this as is for now and maybe align all the diagrams at a later time

**Pull request merged**